packages/cardano-services/test/cli.test.ts

```diff
@@ -2153,30 +2154,67 @@ describe('CLI', () => {
           expect(res.status).toBe(200);
         });
 
-        it('asset provider server', async () => {
-          proc = withLogging(
-            fork(
-              exePath,
-              [
-                ...baseArgs,
-                '--api-url',
-                apiUrl,
-                '--postgres-connection-string-asset',
-                postgresConnectionStringHandle,
-                '--use-typeorm-asset-provider',
-                'true',
-                '--service-names',
-                ServiceNames.Asset
-              ],
-              { env: {}, stdio: 'pipe' }
-            )
-          );
-          await serverStarted(apiUrl);
-          const headers = { 'Content-Type': 'application/json' };
-          const res = await axios.post(`${apiUrl}${services.asset.versionPath}/${ServiceNames.Asset}/health`, {
-            headers
+        describe('asset provider server', () => {
```

Comment:
I like that you've broken `asset provider server` into two `it`'s, wrapped in `describe` :)